spec/plugin.spec.js

```diff
@@ -387,6 +387,28 @@ describe('ManifestPlugin', function() {
       });
     });
 
+    it('ensures the manifest is mapping paths to names', function(done) {
+      webpackCompile({
+        context: __dirname,
+        entry: './fixtures/file.txt',
+        module: {
+          loaders: [
+            { test: /\.(txt)/, loader: 'file-loader?name=[hash].[ext]' },
+          ]
+        }
+      }, {}, function(manifest, stats) {
+        var assetOutputFilename = Object.keys(stats.compilation.assets)[0];
+
+        expect(manifest).toBeDefined();
+        expect(manifest).toEqual({
+          'main.js': 'main.js',
+          'file.txt': assetOutputFilename
```

Comment:
Or instead of a hash, I can use `file-loader?name=outputfile.[ext]` ? What do you think ?